ci-operator/step-registry/mtr/tests/ui/README.md

```diff
@@ -0,0 +1,54 @@
+# mtr-tests-ui-ref<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+
+- [Purpose](#purpose)
+- [Process](#process)
+- [Requirements](#requirements)
+  - [Infrastructure](#infrastructure)
+  - [Variables](#variables)
+  - [Other](#other)
+- [Custom Image - `mtr-runner`](#custom-image---mtr-runner)
+
+## Purpose
+
+Use to execute the `Cypress` [windup-ui-tests](https://github.com/windup/windup-ui-tests) using the provided arguments. All XML results will be combined into "$ARTIFACT_DIR/windup-ui-results.xml".
+
+## Process
+
+1. Retrieves the the test cluster host URL from the `$SHARED_DIR` and uses it to construct the target URL of the MTR webpage in the test cluster.
+2. Executes the Cypress tests using target URL constructed earlier in the script and the `CYPRESS_SPEC` variable
+3. Uses the `npm run mergereports` command to merge all of the XML results into one file.
+4. Copies the XML file from the command in step 4 to `$ARTIFACT_DIR/windup-ui-results.xml`.
+
+## Requirements
+
+### Infrastructure
+
+- A provisioned test cluster to target.
+  - Should have a `mtr` namespace/project with:
+    - [The `mtr-operator` installed](../install/README.md).
```

Comment:
I think this is a broken path to a README should this be `install-operators/README.md?`